extensions/amp-samba-player/0.1/validator-amp-samba-player.protoascii

```diff
@@ -0,0 +1,86 @@
+#
+# Copyright 2016 The AMP HTML Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the license.
+#
+
+tags: {  # amp-samba-player
+  html_format: AMP
+  tag_name: "SCRIPT"
+  spec_name: "amp-samba-player extension .js script"
+  satisfies: "amp-samba-player extension .js script"
+  mandatory_parent: "HEAD"
+  unique_warning: true
+  extension_unused_unless_tag_present: "amp-samba-player"
+  attrs: {
+    name: "async"
+    mandatory: true
+    value: ""
+  }
+  attrs: {
+    name: "custom-element"
+    value: "amp-samba-player"
+    mandatory: true
+    dispatch_key: true
+  }
+  attrs: { name: "nonce" }
+  attrs: {
+    name: "src"
+    mandatory: true
+    value_regex: "https://cdn\\.ampproject\\.org/v0/amp-samba-player-(latest|0\\.1).js"
+  }
+  attrs: {
+    name: "type"
+    value_casei: "text/javascript"
+  }
+  cdata: {
+    blacklisted_cdata_regex: {
+      regex: "."
+      error_message: "contents"
+    }
+  }
+  spec_url: "https://www.ampproject.org/docs/reference/extended/amp-samba-player.html"
+}
+tags: {  # <amp-samba-player>
+  html_format: AMP
+  tag_name: "AMP-SAMBA-PLAYER"
+  disallowed_ancestor: "AMP-SIDEBAR"
+  requires: "amp-samba-player extension .js script"
+  attrs: {
+    name: "data-project-id"
+    mandatory: true
+    value_regex_casei: "[0-9a-f]{32}"
+  }
+  attrs: {
+    name: "data-media-id"
+    value_regex_casei: "[0-9a-f]{32}"
+  }
+  attrs: {
+    name: "data-env"
+    value_regex_casei: "[0-9a-z\\-]"
+  }
+  attrs: {
+    name: "data-protocol"
+    value_regex_casei: "[0-9a-z]{3,5}"
+  }
+  attr_lists: "extended-amp-global"
+  spec_url: "https://www.ampproject.org/docs/reference/extended/amp-samba-player.html"
```

Comment:
The ampproject.org documentation format changed recently. While this would automatically redirect to the new format, lets just use the new format instead:

`spec_url: "https://www.ampproject.org/docs/reference/components/amp-samba-player"`